README.md

```diff
@@ -31,6 +31,12 @@ Install plugins using the `install` command followed by the path to the reposito
 fisher install ilancosman/tide
 ```
 
+To install a plugin from GitLab, prefix `gitlab.com/`.
+
+```console
+fisher install gitlab.com/ainmosni/fish-aws
+```
+
```

Comment:
@ainmosni If you can turn this into one sentence and put it after a `>` (we don't need the example), then I'd be happy to include this in the README.

Reply:
Done.

Reply:
Thank you. I'll think a bit more about it.